.gitignore

```diff
@@ -3,3 +3,5 @@
 /.project
 /pom.xml
 /.settings/
+/dist
```

Comment:
@chiaramasci @davidebz 
Hi, please do not put dist/ in .gitignore. I know that this would be normally best practice, but in our case we need that to distribute our web components. As you might know, our store has a crawler, which takes distributed files from github directly. I will fix that, just as information for you in the future.